src/main/java/de/tum/cit/aet/evaluation/domain/InternalComment.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.cit.aet.evaluation.domain;
+
+import de.tum.cit.aet.application.domain.Application;
+import de.tum.cit.aet.usermanagement.domain.User;
+import jakarta.persistence.*;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "internal_comments")
+public class InternalComment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID internalCommentId;
```

Comment:
Could you explicitly annotate the internalCommentId field with @Column(name = "internal_comment_id", nullable=false) to ensure the column name in the database matches the expected naming convention?
```suggestion
    @Column(name = "internal_comment_id", nullable = false)
    private UUID internalCommentId;
```